plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcTableHandle.java

```diff
@@ -235,6 +236,16 @@ public String toString()
     {
         StringBuilder builder = new StringBuilder();
         builder.append(relationHandle);
+        if (constraint.isNone()) {
+            builder.append(" constraint=FALSE");
+        }
+        else if (!constraint.isAll()) {
+            builder.append(" constraint on ");
+            builder.append(constraint.getDomains().orElseThrow().keySet().stream()
+                    // TODO constraint should be defined on JdbcTableHandle
+                    .map(columnHandle -> ((JdbcColumnHandle) columnHandle).getColumnName())
```

Comment:
should we report actual constraint here?

Reply:
would be nice, but there isn't a good Domain.toString that can be used here